mubench.pipeline/tasks/implementations/publish_findings_task.py

```diff
@@ -97,9 +97,14 @@ def __slice_by_max_files_per_post(self, potential_hits: List[SpecializedFinding]
 
         yield potential_hits_slice
 
-    def _prepare_post(self, finding: SpecializedFinding, version_compile) -> Dict[str, str]:
+    def _prepare_post(self, finding: SpecializedFinding, version_compile, logger) -> Dict[str, str]:
         markdown_dict = self._to_markdown_dict(finding)
-        snippets = finding.get_snippets(version_compile.original_sources_path)
+
+        try:
+            snippets = finding.get_snippets(version_compile.original_sources_path)
+        except:
```

Comment:
`get_snippets` throws a `SnippetUnavailableException` in this case, please catch that particular one. This exception knows the file we couldn't extract a snippet from and also the specific error that occurred. Please mention the finding rank and the file in the warning message, to make this more actionable for users. Also please log the full exception for debugging.